admin_user_reports.php

```diff
@@ -0,0 +1,232 @@
+            <table class="blueTable" style="margin-left: 3%;"">
+            <thead>
+            <tr>
+                <th>Account ID</th>
+                <th>Account Type</th>
+                <th>Account Category</th>
+                <th>Charge</th>
+                <th>Status</th>
+                <th>Email</th>
+                <th>Balance</th>
+                <th>Deactivate</th>
+            </tr>
+            </thead>
+            <tbody id="tableBody">
+            <?php
+            //Reset table to all applications
+            if(isset($_POST['resetTable']))
+            {
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance FROM `1User` ORDER BY isEmployer DESC";
+                $result = mysqli_query($db,$sql);
+            }
+            // See Employers
+            elseif(isset($_POST['seeEmployers']))
+            {
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance 
+                        FROM `1User`
+                        WHERE isEmployer='" . 1 ."'
+                        ORDER BY isEmployer DESC";
+                $result = mysqli_query($db,$sql);
+            }
+            //See JS
+            elseif(isset($_POST['seeJS']))
+            {
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance 
+                        FROM `1User`
+                        WHERE isEmployer='" . 0 ."'
+                        ORDER BY isEmployer DESC";
+                $result = mysqli_query($db,$sql);
+            }
+            // Search by premium option
+            elseif(isset($_POST['filterPremium']))
+            {
+                $category = $_POST['accountCategory'];
+                if($category == 'all' || empty($category))
+                {
+                    $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance FROM oyc353_1.`1User`ORDER BY isEmployer DESC";
+                }
+                else
+                {
+                    $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance 
+                        FROM `1User`
+                        WHERE premiumOpt='" . $category ."'
+                        ORDER BY isEmployer DESC";
+                }
+                $result = mysqli_query($db,$sql);
+            }
+            //See outstanding balance accounts
+            elseif(isset($_POST['seeOutstanding']))
+            {
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance 
+                        FROM oyc353_1.`1User`
+                        WHERE balance<'" . 0 ."'
+                        ORDER BY isEmployer DESC";
+                $result = mysqli_query($db,$sql);
+            }
+            //Default table. All users
+            else{
+                $sql = "SELECT accountID,isEmployer,premiumOpt,charge,status,email,balance FROM oyc353_1.`1User`ORDER BY isEmployer DESC";
```

Comment:
maybe remove the _oyc353_1._ if it is not necessary

Reply:
Done